integrationtest/performance/orm/pom.xml

```diff
@@ -377,7 +377,7 @@
         <repository>
             <id>jboss-public-repository-group</id>
             <name>JBoss Public Maven Repository Group</name>
-            <url>${repository.jboss-public.url}</url>
+            <url>${jboss.public.repo.url}</url>
```

Comment:
This is the perfect example regarding my previous comment :)

Don't you think it would have been better if the property had never been changed in f25b4ed to begin with?

A lot of times a "cleanup" just creates more work for other teams / other components to have to a)figure out why stuff no longer works b) adjust things

Reply:
@Sanne 
Hmmm, you were the one initially suggesting that we should try to improve the consistency with the JBoss world, that's the reason of f25b4ed. I admit it sure would have been better if I hadn't missed a few things when doing it.

Reply:
that's different; having the same version property format has indeed been requested by the productization team, and is useful to compare e.g. with WildFly poms.

Reply:
Yeah, taking a look at it, this thing has been added while I was doing the clean up, that's why I missed them.

Reply:
You might have missed that I used the same variable names than in the JBoss parent pom: https://github.com/jboss/jboss-parent-pom/blob/master/pom.xml#L118

That's why I changed them :).

Reply:
this whole discussion is missing the point :)
I am not questioning the previous patches, I'm just suggesting to be careful with further property renames. Clearly, this commit is a necessary fix.